google/cloud/aiplatform/training_jobs.py

```diff
@@ -3977,16 +4054,35 @@ def _run(
                 Required. The number of data granularity units between data points in the training
                 data. If [data_granularity_unit] is `minute`, can be 1, 5, 10, 15, or 30. For all other
                 values of [data_granularity_unit], must be 1.
+            training_fraction_split (float):
+                Optional. The fraction of the input data that is to be used to train
+                the Model. This is ignored if Dataset is not provided.
+            validation_fraction_split (float):
+                Optional. The fraction of the input data that is to be used to validate
+                the Model. This is ignored if Dataset is not provided.
+            test_fraction_split (float):
+                Optional. The fraction of the input data that is to be used to evaluate
+                the Model. This is ignored if Dataset is not provided.
             predefined_split_column_name (str):
                 Optional. The key is a name of one of the Dataset's data
                 columns. The value of the key (either the label's value or
-                value in the column) must be one of {``TRAIN``,
-                ``VALIDATE``, ``TEST``}, and it defines to which set the
+                value in the column) must be one of {``training``,
+                ``validation``, ``test``}, and it defines to which set the
                 given piece of data is assigned. If for a piece of data the
                 key is not present or has an invalid value, that piece is
                 ignored by the pipeline.
 
                 Supported only for tabular and time series Datasets.
+            timestamp_split_column_name (str):
+                Optional. The key is a name of one of the Dataset's data
+                columns. The value of the key values of the key (the values in
+                the column) must be in RFC 3339 `date-time` format, where
+                `time-offset` = `"Z"` (e.g. 1985-04-12T23:20:50.52Z). If for a
+                piece of data the key is not present or has an invalid value,
+                that piece is ignored by the pipeline.
+
+                Supported only for tabular and time series Datasets.
```

Comment:
@thehardikv How come the proto says `Supported only for tabular Datasets.`. It doesn't include time-series datasets.

Reply:
Like I said in a previous comment, specifying a timestamp split will produce a trained model based on fraction splits.